app/scripts/setup.sh

```diff
@@ -63,10 +63,14 @@ function setupExternalLibraries() {
         echogreen "Checking out the p12n-extract library in $libraries"
         cd $libraries
         git clone https://github.com/flodolo/p12n-extract/ p12n
+        #TODO: remove this
+        cd p12n;git checkout x-channel;cd ..
```

Comment:
There should be no changes to setup.sh

Reply:
hm, looks like GitHub decided it was fine to include those… :/